### PR TITLE
YLP-2046 Migrate table settings from viz to dumb

### PR DESCRIPTION
**Pull request by TimCanu**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

**Comment by TimCanu** [comment]
The component from viz could not be deleted yet as it is used by `HistoryTable` which **inherits** it...

File: packages/dumb/src/components/settings/table.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+export const Table: FunctionComponent<TableProps> = (props) => {
+  const { rows, title } = props
+
+  const { t } = useTranslation()
+
+  const parameterColumnClassName = 'table-diabeloop-parameters-name'
+  const valueColumnClassName = 'table-diabeloop-parameters-value'
+  const unitColumnClassName = 'table-diabeloop-parameters-unit'
```
Comment (fatb38) [inline packages/dumb/src/components/settings/table.tsx]: unused classes, to removed !

File: packages/dumb/src/components/settings/table.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  composes: settingsTable from './settings.css';
+}
+
+.settingsTable .spannedRow td {
+  text-align: center;
+  padding: 0.4em 0;
+  background-color: var(--brand-purple);
+  background-image: linear-gradient(to right, var(--brand-purple-dark) 0, var(--brand-purple-dark) 14px,var(--brand-purple) 14px, var(--white) 100%);
+  color: var(--gray-dark);
+}
```
Comment (fatb38) [inline packages/dumb/src/components/settings/table.css:55]: unused, to remove !

File: packages/dumb/src/components/settings/table.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+@media print {
+  .secondaryLabelWithMain {
+    composes: lightText largeSize from '../../styles/typography.css';
+    color: var(--text-black);
+    composes: secondaryLeftPadding from './settings.css';
+  }
+}
+
+.bdlgSettingsHeader {
```
Comment (fatb38) [inline packages/dumb/src/components/settings/table.css:36]: `bdlg` means blood glucose i guess ? Should be `bdgl` ?

**Review by fatb38 — APPROVED** [review]

**Pull request merged**